package.json

```diff
@@ -15,7 +15,8 @@
     "validate": "npm run lint && npm run build && npm run test && npm run functional && npm outdated --depth 0",
     "validate-dev": "npm run lint && npm run build && npm outdated && npm run test-dev",
     "watch:build": "watch --wait=5 'npm run validate-dev' ./app/",
-    "dev": "npm run watch:build"
+    "dev": "npm run watch:build",
+    "version-healthchecker": "npm run build && node app/version-healthchecker.js"
```

Comment:
We shouldn't run the build in this step. Best practice is to build and validate the build prior to promoting the app to production, so it doesn't make sense to combine the build step and the app start unless you're working on the development.